homeassistant/components/rfxtrx/cover.py

```diff
@@ -118,6 +161,20 @@ async def async_stop_cover(self, **kwargs):
         self._state = True
         self.async_write_ha_state()
 
+    async def async_open_cover_tilt(self, **kwargs):
```

Comment:
We need to update the supported features for the entity too when adding tilt support.

Reply:
Done.

Reply:
I guess I went the wrong way to implement tilt support.  Changed the implementation.